cpp/common/test/rules/validcontainerelementaccess/test.cpp

```diff
@@ -96,14 +96,14 @@ void f8(const int *ar) {
 void f9(const std::string &s, std::string &str) {
   std::string::iterator loc = str.begin();
   for (auto i = s.begin(), e = s.end(); i != e; ++i, ++loc) { // NON_COMPLIANT
-    str.insert(loc, 'c');                                     // NON_COMPLIANT
+    str.insert(loc, 'c'); // NON_COMPLIANT[FALSE POSITIVE for str]
```

Comment:
```suggestion
    str.insert(loc, 'c'); // NON_COMPLIANT[FALSE POSITIVE] - false positive for str
```

Reply:
Oh, I fixed the false positives in this file - let me revert this change.